### PR TITLE
Add backup settings and final snapshot flag

### PR DESCRIPTION
**Pull request by gstlt**
Hope this helps if someone needs a backup of the DB :-)

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -33,6 +33,13 @@ resource "aws_db_instance" "main_rds_instance" {
     # Upgrades
     allow_major_version_upgrade = "${var.allow_major_version_upgrade}"
     auto_minor_version_upgrade  = "${var.auto_minor_version_upgrade}"
+
+    # Snapshots and backups
+    skip_final_snapshot = "${var.skip_final_snapshot}"
+    copy_tags_to_snapshot = true
```
Comment (antonbabenko) [inline main.tf]: make `copy_tags_to_snapshot` and `backup_window` as variables also, please.

Comment (gstlt) [reply main.tf]: Will do. Thanks!

**Comment by gstlt** [comment]
Anything more I should change here?

BTW: There's bug in Terraform regarding this option: https://github.com/hashicorp/terraform/issues/5417

**Comment by antonbabenko** [comment]
Sorry for the delay. Merging now.

**Pull request merged**